builtin/energy/energy.go

```diff
@@ -285,7 +285,7 @@ func (e *Energy) DistributeRewards(beneficiary, signer thor.Address, staker stak
 
 	// If delegated amount of VET is 0 then transfer the whole reward to the validator
 	proposerReward := new(big.Int).Set(reward)
-	if hasDelegations {
+	if hasDelegations && validatorRewardPerc.Cmp(big.NewInt(100)) < 0 {
```

Comment:
Does this mean that if a validator has delegators but the `validatorRewardPerc` for some reason is greater or equal 100, delegators are not getting any rewards?